pkg/gofr/http/router.go

```diff
@@ -14,6 +17,14 @@ type Router struct {
 	RegisteredRoutes *[]string
 }
 
+type StaticFileConfig struct {
+	DirectoryListing  bool
+	HideDotFiles      bool
```

Comment:
Is there still a need for this variable then?

Reply:
Not required thanks for highlighting.